cmake/sphinx.cmake

```diff
@@ -169,8 +169,20 @@ endmacro()
 #   Please refer to the *Sphinx* paragraph in the *General Documentation* in this
 #   package for more explanation about the parametrization of the tools.
 #
+#   Optional Arguments:
+#
+#   - DOXYGEN_EXCLUDE_PATTERNS:  String that is added to EXCLUDE_PATTERNS in the
```

Comment:
what format should this string have ?

Reply:
I added more explanation with an example.